src/main/java/Dragons/dragonTypes.java

```diff
@@ -0,0 +1,5 @@
+package Dragons;
+
+public enum dragonTypes {
```

Comment:
Класи ми називаємо `CamelCase` та не через plural i.e. just `DragonType`.
```suggestion
public enum DragonType {
```